httpd.go

```diff
@@ -50,7 +53,21 @@ func (h *Httpd) Run() {
 
 func messageHandler(p Param) (int, string) {
 	ch := make(chan error, 1)
-	go MessageBus.Publish(NewMessage(p, ch))
+
+	// The format is compat with JavaScript's date.toISOString().
```

Comment:
This format is an extension of ISO8601, right? If so, the format may be unfamiliar with other languages. Any other alternative?